test/python/expected/upgrade_validation/expected_dependency.out

```diff
@@ -655,6 +655,7 @@ Function sys.tsql_type_max_length_helper(text,integer,integer,boolean,boolean)
 Function sys.tsql_type_precision_helper(text,integer)
 Function sys.tsql_type_radix_for_sp_columns_helper(text)
 Function sys.tsql_type_scale_helper(text,integer,boolean)
+Function sys.typeproperty(sys."varchar",sys."varchar")
```

Comment:
An entry here indicates missing dependency test. Please add the same.

Reply:
Added dependency test case.